### PR TITLE
chore(space-wizard): refactoring

### PR DESCRIPTION
**Pull request by mitchharpur**
- more refactoring of space wizard into smaller components
     - multi select wizard widget
     - single select wizard widget
     - some widget renaming to better suite function
- provided access to forge create quick start project command for multiselect widget demonstration
- few small defects found and fixed

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 41ce3cbec18a02f1cf36639a1b9439735da88672
  * Your subject line is longer than 50 characters

 * Commit: ed9b33b567b1d246720cce6e18cc0b4236dcd47a
  * Your subject line is longer than 50 characters

 * Commit: dc7066932687367ab9564c049bfd0a02a2976ee4
  * Your subject line is longer than 50 characters

 * Commit: 9468cba878792321062a782da391fbf946b95596
  * Your subject line is longer than 50 characters

 * Commit: 47424312af09eee709da5c781fd638d6225c800e
  * Your subject line is longer than 50 characters

 * Commit: edb16b196852719096326e4c3d905d572ae69f58
  * Your subject line is longer than 50 characters

 * Commit: 2ecc22a15b779a327c9e212b99f68bca27b662db
  * Your subject line is longer than 50 characters

 * Commit: 8f1fa77dc1e54b82be946666f4472a6e4e96764d
  * Your subject line is longer than 50 characters

 * Commit: ea992010df9bb98f896b90b8c69d94c678489e7f
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@mitchharpur snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1439-fabric8-ui.int.rdu2c.fabric8.io

**Comment by joshuawilson** [comment]
[test]

File: src/app/space-wizard/services/concrete/fabric8-app-generator-configurator.service.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ export class AppGeneratorConfiguratorService {
     this.context.current.subscribe( (ctx: Context) => {
       if (ctx.space) {
         this.currentSpace = ctx.space;
-        this.log(`the current space is updated to ${this.currentSpace.attributes.name}`);
+        this.log(`The current spacehas been updated to ${this.currentSpace.attributes.name}`);
```
Comment (joshuawilson) [inline src/app/space-wizard/services/concrete/fabric8-app-generator-configurator.service.ts]: There is a typo here. "spacehas"

**Pull request merged**